### PR TITLE
docs: adding copy for ubuntu 20.04 workaround.

### PR DESCRIPTION
**Pull request by jasontclark**
Signed-off-by: Jay Clark <jay@convox.com>

### CODE REVIEW

**Review by kaiomagalhaes — APPROVED** [review]

File: docs/installation/development-rack/ubuntu.md [hunk 1]
```diff
@@ -2,12 +2,40 @@
 
 ## Initial Setup
 
-> **_NOTE:_**:  Currently, we guarantee support for Ubuntu up to version 18.04. Though newer versions of Ubuntu have yet to be fully supported, we are actively working on support for Ubuntu version 20.04.
+> **_NOTE:_**:  Ubuntu 20.04 users will need ensure that they are installing the latest version of convox. See more 'Tips' sections below for more details.
```
Comment (beastawakens) [inline docs/installation/development-rack/ubuntu.md:5]:
```suggestion
> **_NOTE:_**:  Ubuntu 20.04 users will need to ensure that they are installing the latest version of convox. See more 'Tips' sections below for more details.
```

File: docs/installation/development-rack/ubuntu.md [hunk 2]
```diff
@@ -2,12 +2,40 @@
...
 
 ### Docker
 
     $ sudo apt install docker.io
 
+---
+
+### Tips for Ubuntu 20.04 Users
+
+As an Ubuntu 20.04 user, in order to get the Kubernetes steps that follow to work, a slight modification to your local `/etc/resolv.conf` file is required. `microk8s` leverages your local DNS configuration to configure DNS resolution in Kubernetes, and the following commands will provide a workaround to a bug that is encountered with Kubernetes version 1.13:
```
Comment (beastawakens) [inline docs/installation/development-rack/ubuntu.md:15]:
```suggestion
As an Ubuntu 20.04 user, in order to get the Kubernetes steps that follow to work, a slight modification to your local `/etc/resolv.conf` file is required. `microk8s` leverages your local DNS configuration to configure DNS resolution in Kubernetes, and the following commands will provide a workaround to a bug that is encountered with microk8s version 1.13:
```

File: docs/installation/development-rack/ubuntu.md [hunk 3]
```diff
@@ -36,6 +64,33 @@ Install a local Rack named `dev`.
...
+    Name      dev
+    Provider  local
+    Router    router.dev.convox
+    Status    running
+    Version   3.0.49
+
+If it doesn't, most likely you will receive a `504` error, as your local firewall rules are not allowing traffic to be forwarded to microk8s.
+This can be resolved with the following command:
+
+  $ iptables -P FORWARD ACCEPT
```
Comment (beastawakens) [inline docs/installation/development-rack/ubuntu.md:88]:
```suggestion
    $ iptables -P FORWARD ACCEPT
```

**Review by beastawakens — CHANGES_REQUESTED** [review]

File: docs/installation/development-rack/ubuntu.md (hunk 1 above)
Comment (jasontclark) [reply docs/installation/development-rack/ubuntu.md:5]: 👍

File: docs/installation/development-rack/ubuntu.md (hunk 2 above)
Comment (jasontclark) [reply docs/installation/development-rack/ubuntu.md:15]: 👍

File: docs/installation/development-rack/ubuntu.md (hunk 3 above)
Comment (jasontclark) [reply docs/installation/development-rack/ubuntu.md:88]: 👍

**Review by beastawakens — DISMISSED** [review]
:shipit:

File: docs/installation/development-rack/ubuntu.md [hunk 4]
```diff
@@ -2,12 +2,39 @@
 
 ## Initial Setup
 
-> **_NOTE:_**:  Currently, we guarantee support for Ubuntu up to version 18.04. Though newer versions of Ubuntu have yet to be fully supported, we are actively working on support for Ubuntu version 20.04.
+> **_NOTE:_**:  Ubuntu 20.04 users will need ensure that they are installing the latest version of convox. See more 'Tips' sections below for more details.
```
Comment (ddoc) [inline docs/installation/development-rack/ubuntu.md]: Capitalize Convox Please

**Review by beastawakens — APPROVED** [review]
:shipit:

**Pull request merged**